python_modules/dagster/dagster/_core/definitions/schedule_definition.py

```diff
@@ -55,19 +55,20 @@
 if TYPE_CHECKING:
     from dagster import ResourceDefinition
     from dagster._core.definitions.repository_definition import RepositoryDefinition
+    from dagster._core.definitions.run_config import RunConfig
 T = TypeVar("T")
 
-RunConfig: TypeAlias = Mapping[str, Any]
+RunConfigType: TypeAlias = Union["RunConfig", Mapping[str, Any]]
```

Comment:
Thoughts on `CoercibleToRunConfig`? We don't typically use `Type` as a suffix in these kinds of situations.

Reply:
funny you say that because we have that type defined elsewhere already 😄 

updated